apps/nowcasting-app/components/map/pvLatest.tsx

```diff
@@ -199,15 +199,15 @@ const PvLatestMap = () => {
     <FailedStateMap error="Failed to load" />
   ) : forecastLoading ? (
     <LoadStateMap>
-      <ButtonGroup rightString={formatISODateStringHuman(selectedISOTime || "")} />
+      <ButtonGroup rightString={formatISODateStringHuman(selectedISOTime || " ")} />
     </LoadStateMap>
   ) : (
     <Map
       loadDataOverlay={addFCData}
       updateData={{ newData: !!initForecastData, updateMapData }}
       controlOverlay={(map: { current?: mapboxgl.Map }) => (
         <>
-          <ButtonGroup rightString={formatISODateStringHuman(selectedISOTime || "")} />
+          <ButtonGroup rightString={formatISODateStringHuman(selectedISOTime || " ")} />
```

Comment:
Are these new spaces intentional here and above on line 202? No worries if they are necessary, just checking wasn't a weird linting thing

Reply:
this wasn't intentional